contracts/Makefile

```diff
@@ -1,4 +1,5 @@
-RPC_URL ?= http://localhost:8545
+RPC ?= http://localhost:8545
+PK ?= 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

Comment:
This is just the private key for the first account of the `test test test ... test junk` mnemonic.